### PR TITLE
Update to RN 58

### PR DESCRIPTION
**Pull request by acoates-ms**

### CODE REVIEW

File: RNWCPP/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-windows",
-  "version": "0.57.0-vnext.30",
+  "version": "0.58.0-vnext.0",
```
Comment (HeyImChris) [inline RNWCPP/package.json]: "6" - Is there a reason we wouldn't just make this the same version as the fb one we pull in?

Comment (acoates-ms) [reply RNWCPP/package.json]: This version will bump independently, based on changes made to RNW.

**Pull request merged**